src/app/components/CityCard/CityCard.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import ButtonSmall from '../ButtonSmall/ButtonSmall';
+import styles from './CityCard.module.css';
+
+type CityCardProps = {
+  locationName: string;
+  region: string;
+};
+
+export default function CityCard({
+  locationName,
+  region,
+}: CityCardProps): JSX.Element {
+  return (
+    <section className={styles.cityCard}>
```

Comment:
`article` would be more fitting in my opinion, it's a common discussion on the web though :D